recipes/conda-workon/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "conda-workon" %}
+{%set version = "0.3.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "af1ee8f744c5379afa0c2b8c4156a2c0bb282fe6461cc2b087c2b789c8bb90d3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  entry_points:
+    - conda-workon = conda_workon:main
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - conda
+
+  run:
+    - python
+    - conda
+
+test:
+  commands:
+    - conda workon --help
+    - conda-workon --help
+
+about:
+  home: https://bitbucket.org/flub/conda-workon
+  license: BSD (Clause unspecified)
```

Comment:
See [here](https://bitbucket.org/flub/conda-workon/issues/3/type-of-bsd-license)